WordPress/Classes/ViewRelated/Post/Search/PostSearchService.swift

```diff
@@ -81,7 +81,9 @@ final class PostSearchService {
             offset += posts.count
             hasMore = !posts.isEmpty
 
-            let newPosts = posts.filter { !postIDs.contains($0.objectID) }
+            let newPosts = posts
```

Comment:
The existing defensive code was not enough to protected from the scenario when there are duplicated within a single response (`posts`). This is what I assume is causing the crash.